Jenkinsfile.integration

```diff
@@ -0,0 +1,33 @@
+node('vetsgov-general-purpose') {
+  // Checkout vets-website code
+  dir("vets-website") {
+    checkout scm
+    ref = sh(returnStdout: true, script: 'git rev-parse HEAD').trim()
+  }
+
+  def commonStages = load "vets-website/jenkins/common.groovy"
+
+  // Setup stage
+  dockerContainer = commonStages.setup()
+
+  stage('Integration') {
+    dir("vets-website") {
+      try {
+        parallel (
+          nightwatch: {
+            sh "export IMAGE_TAG=${commonStages.IMAGE_TAG} && docker-compose -p e2e up -d && docker-compose -p e2e run --rm --entrypoint=npm -e BABEL_ENV=test -e BUILDTYPE=vagovprod -e WEB_HOST=staging.va.gov -e WEB_PORT=80 vets-website --no-color run nightwatch:docker -- --tag integration"
```

Comment:
I think we'll need to update the `Access-Control-Allow-Origin` header.

In some cases like the MegaMenu and (I think) footer, I think we use absolute URLs for TeamSite compatibility. I'm not sure if we have E2E tests that may be affected by that, and may end up going to the actual Staging website.

Reply:
These test aren't the teamsite tests, these are nightwatch tests that I'm pointing to the actual staging site so that they run against it and call the live staging api. This works without much tweaking for forms e2e tests, will probably require more work for the tests for other apps.

I would eventually like to run the teamsite tests, too, but those are commented out below. They would be run against prod or staging, though.

Reply:
Oh, completely misunderstood that this is the actual Staging site we're executing against. Okay, makes sense 👌